src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -180,7 +187,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $errorIo->comment('To search for a specific tag, re-run this command with a search term. (e.g. <comment>debug:container --tag=form.type</comment>)');
             } elseif ($input->getOption('parameters')) {
                 $errorIo->comment('To search for a specific parameter, re-run this command with a search term. (e.g. <comment>debug:container --parameter=kernel.debug</comment>)');
-            } else {
+            } elseif (!$input->getOption('deprecations')) {
```

Comment:
The `validateInput` method below should be updated to take the new option into account.

Reply:
Oh right ! Fixed in the new version :)